outline/connectivity.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package connectivity
+package outline
```

Comment:
I think this will break the old CheckConnectivity

Reply:
This will not break Outline Client, it is using `shadowsocks.CheckConnectivity`: [Android](https://github.com/Jigsaw-Code/outline-client/blob/c29197957a6cb2ea84ba4d619d97f4d836f98094/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java#L370), [Apple](https://github.com/Jigsaw-Code/outline-client/blob/c29197957a6cb2ea84ba4d619d97f4d836f98094/src/cordova/apple/OutlineAppleLib/Sources/PacketTunnelProviderSources/PacketTunnelProvider.m#L141). But you are right, it might break other 3rd party users.